src/index.ts

```diff
@@ -0,0 +1,10 @@
+import getSdk = require('resin-sdk');
+import settings = require('resin-settings-client');
+
+export = getSdk({
```

Comment:
Yeah. I've tried ES6 from TS and I had to explicitly use `.default` in the consumer non-TS modules.
But afaik there's a plugin for Webpack that can normalize it (by checking the __esModule prop on the exported object).